front/src/components/song-type-icon.tsx

```diff
@@ -0,0 +1,69 @@
+/*
+ * Meelo is a music server and application to enjoy your personal music files anywhere, anytime you want.
+ * Copyright (C) 2023
+ *
+ * Meelo is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * Meelo is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import { Box, Tooltip } from "@mui/material";
+import { useTranslation } from "react-i18next";
+import { type SongType } from "../models/song";
+import {
+    SongIcon,
+    SongTypeAcapellaIcon,
+    SongTypeAcousticIcon,
+    SongTypeCleanIcon,
+    SongTypeDemoIcon,
+    SongTypeEditIcon,
+    SongTypeInstrumentalIcon,
+    SongTypeLiveIcon,
+    SongTypeMedleyIcon,
+    SongTypeNonMusicIcon,
+    SongTypeOriginalIcon,
+    SongTypeRemixIcon
+} from "./icons";
+
+const SongTypeIcon = ({ type, size = 20 }: { type: SongType; size?: number }) => {
```

Comment:
I think we can remove that component: the tooltip is not useful in both places it's used in, and I don't like hard-coding paddings and positions. What you can do instead is something like:

```ts
const SongTypeIcon = ({ type, ...props }: {type: SongType } & IconProps) => {
    const Icon = () => { 
       switch (type) {
           case 'Original': return SongTypeOriginalIcon;
           // Do the other cases
       }
    }
    return <Icon {...props}/>
}
```

The paddings and alignments should be done at the places where you use the icons